lib/loadTransforms.js

```diff
@@ -3,15 +3,7 @@ import { transformsPath } from "./scaffold";
 
 module.exports = function loadTransforms() {
   const transformsFilePath = transformsPath();
-  return new Promise((resolve, reject) => {
-    let transforms = [];
-    try {
-      delete require.cache[require.resolve(transformsFilePath)];
-      const factory = require(transformsFilePath);
-
-      Promise.resolve(factory()).then(resolve).catch(reject);
-    } catch (error) {
-      reject(error);
-    }
-  });
+  delete require.cache[require.resolve(transformsFilePath)];
+  const factory = Module._load(transformsFilePath, module, false);
+  return factory();
```

Comment:
In order to return a rejected Promise in the case of a synchronous error, we need to keep the Promise boilerplate I had here. Otherwise, looks good

Reply:
@suchipi oh whoops, I meant to make the function async, which would solve that problem